src/components/Header.js

```diff
@@ -1,99 +1,101 @@
 import React from 'react'
 import { Link } from 'gatsby'
 import { css } from '@emotion/core'
-import theme from '../../config/theme'
+import { useTheme } from '../../config/theming'
```

Comment:
I'd consider putting this in /components instead of config...

Cleaner import and since it will be used quite a bit...

Reply:
Gotcha, will fix.